### PR TITLE
Blog/2017-08-28: load thresholding

### PR DESCRIPTION
**Pull request by mfuhrmann**
A legere blog about cpu load thresholds
Atm the wiki page is still missing. So please do not merge in this state.

### CODE REVIEW

File: user/pages/05.blog/2017-08-15-load-thresholding/item.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+I found my favorite one on [blog.scoutapp.com](http://blog.scoutapp.com/articles/2009/07/31/understanding-load-averages).
+
+When you have read this article you know, that the maximum load on a Linux machine depends on the amount of CPU cores.
+And maybe you already foresee what that means for your threshold configs.
+Yes, you will need individual thresholds for your servers depending on the amount of CPU cores.
+
+And... wait.. Accidentally we have some "ready to use" configurations on the wiki.
+Lucky bastard! :-)
+
+Would you like to know more? link to wiki
```
Comment (mvrueden) [inline user/pages/05.blog/2017-08-15-load-thresholding/item.md]: The link is missing.

File: user/pages/05.blog/2017-08-15-load-thresholding/item.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    tag: [configuration, thresholding]
+---
+
+"System has high load!"
+
+Sounds everything but perfect.
+At least when we are talking about servers.
+
+Usual Windows user never heard about that because it's just a Linux thingy.
+Maybe that's not totally correct, but at least Windows doesn't have a similar metric.
```
Comment (mvrueden) [inline user/pages/05.blog/2017-08-15-load-thresholding/item.md]: The section about windows is totally irrelevant to the blog post I think. Maybe we put it to the bottom or somewhere lower than the top of the article?

File: user/pages/05.blog/2017-08-15-load-thresholding/item.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    category: wiki
+    tag: [configuration, thresholding]
+---
+
+"System has high load!"
+
+Sounds everything but perfect.
+At least when we are talking about servers.
+
+Usual Windows user never heard about that because it's just a Linux thingy.
```
Comment (SISheogorath) [inline user/pages/05.blog/2017-08-15-load-thresholding/item.md]: "~Linux~ thing~y~" -> "Unix-thing"

File: user/pages/05.blog/2017-08-15-load-thresholding/item.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Maybe that's not totally correct, but at least Windows doesn't have a similar metric.
+
+Fortunately I don't have to explain what load is or how it's defined because there are many explanation in the internet.
```
Comment (SISheogorath) [inline user/pages/05.blog/2017-08-15-load-thresholding/item.md]: "Fortunately,"

"*on* the internet"

File: user/pages/05.blog/2017-08-15-load-thresholding/item.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+I found my favorite one on [blog.scoutapp.com](http://blog.scoutapp.com/articles/2009/07/31/understanding-load-averages).
+
+When you have read this article you know, that the maximum load on a Linux machine depends on the amount of CPU cores.
```
Comment (SISheogorath) [inline user/pages/05.blog/2017-08-15-load-thresholding/item.md]:
```
Once you read the article, you know that the ideal load on a Linux machine depends on the number of CPU cores.
```

There is no maximum load.

**Review by SISheogorath — COMMENTED** [review]
Only some minor corrections.

Should be reviewed by a native speaker before publishing.

**Comment by mfuhrmann** [comment]
thanks  @SISheogorath

**Comment by mfuhrmann** [comment]
@mvrueden If you want, you can merge this next week. Monday would be perfect.

**Comment by mfuhrmann** [comment]
@JessiDoesStuff @jeffgdotorg Would be nice if you could read it and add some sprinkles. Thanks!

**Comment by JessiDoesStuff** [comment]
@jeffgdotorg maybe should look over my shoulder? There's a few technical things I don't understand D:

**Comment by jeffgdotorg** [comment]
Looks technically good to me.

@mfuhrmann maybe a topic for a future article: it's really easy to use Net-SNMP's `extend` directive to create a scalar MIB object containing the CPU count. On Linux, just have it run `/bin/egrep -c ^processor /proc/cpuinfo`; configure OpenNMS to collect it from whatever OID gets assigned, and start using the value in your expression-based thresholds.

**Comment by mfuhrmann** [comment]
@jeffgdotorg Indeed. It's a cool thing. I will give it a try.

**Review by mvrueden — APPROVED** [review]

**Pull request merged**